MSAL/test/unit/MSALDeviceInfoProviderTests.m

```diff
@@ -90,6 +91,56 @@ - (void)testGetDeviceInfo_whenCurrentSSOExtensionRequestAlreadyPresent_shouldRet
     [self waitForExpectations:@[expectation, failExpectation] timeout:1];
 }
 
+- (void)testWPJMetaDataDeviceInfoWithRequestParameters_tenantIdNil API_AVAILABLE(ios(13.0), macos(10.15))
```

Comment:
Can you add an error scenarios as well? e.g. keychain reading fails or item not found